docs/manual/exporting-the-schedule/index.md

```diff
@@ -352,11 +352,11 @@ The &lt;div&gt; element for this widget would be:
 
     <div id="scheduleTabs"></div>
 
-Using the code above and the CSS and image files provided with Airtime, the weekly schedule can be styled to look like this:
+Using the code above and the CSS and image files provided with Libretime, the weekly schedule can be styled to look like this:
 
 ![](static/Screenshot-87-weekly_schedule_widget.png)
 
-Optionally, you can display the schedule for the week ahead, as well as the current week. This requires the Javascript file *jquery.showinfo.js* supplied with Airtime 2.5.0 or later, where the value of the variable *dow* (for days of the week) includes fourteen rather than seven days:
+Optionally, you can display the schedule for the week ahead, as well as the current week. This requires the Javascript file *jquery.showinfo.js* supplied with Libretime 2.5.0 or later, where the value of the variable *dow* (for days of the week) includes fourteen rather than seven days:
```

Comment:
@hairmare is `jquery.showinfo.js` still supplied?

Reply:
It's here: https://github.com/LibreTime/libretime/blob/master/airtime_mvc/public/widgets/js/jquery.showinfo.js

I'm not sure if anyone tested it and i suspect it might need quite some customization if you would want to use it on your station.